compiler/test/dotty/tools/vulpix/ParallelTesting.scala

```diff
@@ -672,6 +675,35 @@ trait ParallelTesting extends RunnerOrchestration { self =>
       (errorMap, expectedErrors)
     }
 
+    /**
+     * We want to warn the developer about some of the possible typos when trying to point "error" and "nopos-error"
+     * For example - A warning could be emitted if the developer mistakenly types in "//error" instead of "// error"
+     * or "//nopos-error" instead of "// nopos-error"
+     *
+     * @param files test files
+     * @return Sequence of warning messages
+     */
+    def getWarnings(files : Seq[JFile]) : Seq[String] = {
```

Comment:
We should check this in `getErrorMapAndExpectedCount` and emit the warnings in it. The main motivation is to process each file/line once.

Reply:
I would suggest performing the check here:

https://github.com/lampepfl/dotty/pull/7062/files#diff-3be4e64955ddae8ae1de36d6f1fdc1bbR670

Reply:
done